### PR TITLE
Active records in rails

### PR DESCRIPTION
**Pull request by karthykarthick**
in this i implemented the active records with the help of the readings

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,7 +9,6 @@
 
 👤 **karthick harimoorthy**
```
Comment (christianotieno) [inline README.md:17]: # Improve on the README
 - [ ] The project's README has to be as descriptive as possible ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)). KIndly add instruction on here on how to well clone and run the code.

File: test/channels/application_cable/connection_test.rb [hunk 2]
```diff
@@ -1,4 +1,4 @@
-require "test_helper"
+require 'test_helper'
```
Comment (christianotieno) [inline test/channels/application_cable/connection_test.rb:2]: # Improve on the commit messages
- [  ] Your commit messages aren't as clear as they are expected on this project. In your next round of changes, let's improve on this area. thank you. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_commits.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)).

**Review by christianotieno — CHANGES_REQUESTED** [review]
# Changes Required ♻️

Hello team, good job with the logic. It looks solid. Keep it up. 👏 
Before we close this let's take care of some issues below 👇  

### Happy coding 💻 
#### Christian Otieno (TSE)

**Review by tirthajyoti-ghosh — APPROVED** [review]
# Approved ✅

## You have done an incredible job making this project 👏🧨💪! Just perfect 💯! 

### You have met all the requirements of the project. Go ahead and merge it! Onto the next project 🚀 !

Happy coding 👨‍💻 

<hr>

**Tirthajyoti Ghosh**

[GitHub](https://github.com/tirthajyoti-ghosh) | [LinkedIn](https://www.linkedin.com/in/tirthajyoti-ghosh/) | [Twitter](https://twitter.com/terrific_ghosh)

**Pull request merged**